### PR TITLE
[Enhancement] Mostly RHEL-7 (but touching also other products) -- Add selected OVAL checks implementation and one remediation

### PR DESCRIPTION
**Pull request by iankko**
This changeset is performing the following:
* patch https://github.com/OpenSCAP/scap-security-guide/commit/2f7111cb8085ebc99a42e9c65118b2c9867515ef -- ads new ```shared/``` version of the OVAL check for ```account_unique_name``` rule,
* patch https://github.com/OpenSCAP/scap-security-guide/commit/5d4839672cf9592c6e0aa1d0d3b209863f72bbd7 -- enhances the ```Makefile`` for the ```RHEL/5``` product to look into the ```shared/oval``` directory too (so selected existing, currently RHEL-5 product specific OVAL checks can be moved to ```shared/oval``` directory and re-used for multiple products [after appropriate changes]),
* patch https://github.com/OpenSCAP/scap-security-guide/commit/632b42d57753c9e2006eb7ad70c7d2c043860fc0 -- is modifying the existing ```RHEL/5``` product specific currently OVAL check for ```gid_passwd_group_same``` rule to be re-usable for more products, and moving it into the ```shared/oval``` folder. Fixes couple of trailing whitespace noise issues in the XCCDF descriptions for this rule across different products (where the fix was applicable),
* patch https://github.com/OpenSCAP/scap-security-guide/commit/b62c1e934bc651f2e854502dad5ef7aff6d47fd9 -- is modifying currently RHEL-5 specific OVAL check for ```aide_build_database``` rule to be more generic and re-usable for multiple products (```RHEL/6, RHEL/7```, and ```Fedora``` too). Also moves that OVAL into ```shared/oval``` directory,
* and finally patch https://github.com/OpenSCAP/scap-security-guide/commit/5edc47b0b720edda0a1702ce2a9d6b282c760d17 -- is adding new ```RHEL/7``` product specific remediation script for ```aide_build_database``` rule. For now it needs to ```RHEL/7``` product specific, since paths to ```aide``` executable differ across different products. In the future we might enhance it yet to check which product we are currently running at, and based on that use appropriate path to the ```aide``` binary in the remediation. But this would be future enhancement.

Testing report:
--------------------
All of the changes have been tested on the following three systems:
* RHEL-6,
* RHEL-7, and
* Fedora 22

and all of them seem to be working fine (return PASS for cases when they should return PASS, and FAIL otherwise) as far as I can tell from testing.

Please review.

Thank you, Jan.

### CODE REVIEW

**Comment by iankko** [comment]
Ad the failing Jenkins - it's not a problem of this particular pull request. It's rather problem that current master fails when run on RHEL-6 system.

All the following three results are identical:
[1] https://jenkins.open-scap.org/job/scap-security-guide-pull-requests/92/console
[2] https://jenkins.open-scap.org/job/scap-security-guide-pull-requests/91/console
[3] https://jenkins.open-scap.org/job/scap-security-guide-pull-requests/93/console

in other words the faiing Jenkins on those is not problem of those pull requests, but rather problem of currently failing ```master```.

**Comment by shawndwells** [comment]
make works on rhel6 and 7.  ./testcheck passes on gid check. merging. thanks @iankko !
